cogs/helpermessage.py

```diff
@@ -0,0 +1,144 @@
+import discord
+import toml
+from discord.ext import commands
+from utils.commands import available_subcommands
+from utils.pagination import paginated_embed_menus, PaginationView
+
+class Helpermessage(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.data = toml.load('config.toml')
+        self.subjects = self.data['subjects']
+        self.helpermessages = self.data['helpermessages']
+        self.subject_channels = self.subjects.keys()
+        self.helper_roles = [self.subjects[channel] for channel in self.subject_channels]
+
+    @commands.hybrid_group()
+    async def helpermessage(self, ctx: commands.Context):
+        """
+        Commands for handling reminders.
+        """
+        await available_subcommands(ctx)
+
+    @helpermessage.command()
+    async def list(self, ctx: commands.Context):
+        """
+        List all active helpermessage embeds
+        """
+        embed_dict = dict(
+            title = 'List of all active helpermessages',
+            description = f'Here is a list of all active helpermessages.',
+        )
+        names = [f'<#{channel}>' for channel in self.helpermessages.keys()]
+        values = [f'https://discord.com/channels/{ctx.guild.id}/{channel}/{int(self.helpermessages[channel])}' for channel in self.helpermessages.keys()]
+        embeds = paginated_embed_menus(names, values, embed_dict=embed_dict)
+        embed, view = await PaginationView(ctx, embeds).return_paginated_embed_view()
+        await ctx.send(embed=embed, view=view)
+
+    async def embed_getter(self, edited_roles):
+        for role in edited_roles:
+            for channel, roles in self.subjects.items():
+                if isinstance(role, list):
```

Comment:
Is this meant to be `isinstance(roles, list)`?

In any case, I think this is a little messy. Ideally `roles` should always be a collection, so `role.id in roles` is a valid predicate in the single and multi-relation cases. You could change the structure of the `config.toml` to suit this, or preprocess the `self.subjects` attribute in a suitable way in your `__init__`:
```py
self.subjects = {
    channel: item if isinstance(item, list) else [item]
    for channel, item in self.subjects.items()
}
```